### ISSUE TITLE
Data Usage Remaining missing on some family accounts

### ISSUE THREAD

**Issue by kw4c** [ryanmac8/HA-Mint-Mobile#12]
**Describe the bug**
I have three family accounts connected to my main account. Two of the family accounts report Data Usage Remaining as unavailable. One is a 35GB "unlimited" account which accounts up the data instead of counting down to zero data. I suspect that might be causing that issue. The second family account has 10GB of data which is odd because the third family account is also 10GB and it works fine. 

<img width="304" alt="Screen Shot 2021-06-04 at 12 25 27 AM" src="https://user-images.githubusercontent.com/31724374/120750436-f99aa980-c4cb-11eb-814f-8b198d5bb8e4.png">
**To Reproduce**
n/a

**Expected behavior**
It would show the data remaining for the 10GB and either show remaining data or used data on the 35GB account.

**Desktop (please complete the following information):**
 - Latest Hassio in a VM

**Comment by ryanmac8** [comment]
Since I don't have unlimited data, I will need for you to inspect the Mint Mobile webpage so I see what values are being passed around to fix this. I have a feeling the unlimited plan is causing the issue with the 10GB plan, so i'd like to fix that issue first.  If you use chrome, you can inspect the webpage and look under the network section for the following API requests.

- Under the Mint Mobile Family page, there's an request to https://w3b-api.ultramobile.com/v1/mint/account/xxxxxxxx/multi-line?. Can you please provide the json response for what's under the current plan.
- Under the same webpage, you should see a API call to https://w3b-api.ultramobile.com/v1/mint/account/xxxxxxxx/multi-line/xxxxxxxx/usage?. Can you provide the json response.

**Comment by kw4c** [comment]
It appear more than two in a family is the issue. I removed two, deleted the integration, rebooted and reinstalled, and setup my account. That was fine. Once installed a third account, didn't matter which, it would go wonky again. I've now added each of the secondary accounts as unique integration instants then added my primary account integration. I can not see all four.

I'd rather not share the json responses. It has phone numbers, codes and unique ids which I'm not comfortable exposing. Here is what the unlimited looks like when it's working. It's counting to down from 35GB after all Unlimited is myth in mobile.
<img width="277" alt="Screen Shot 2021-06-13 at 12 52 47 AM" src="https://user-images.githubusercontent.com/31724374/121796980-cb5f4d00-cbe2-11eb-996e-2c1a132051ca.png">
